website/docs/r/ram_resource_share_accepter.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "aws"
+page_title: "AWS: aws_ram_resource_share_accepter"
+sidebar_current: "docs-aws-resource-ram-resource-share-accepter"
+description: |-
+  Manages accepting a Resource Access Manager (RAM) Resource Share invitation.
+---
+
+# Resource: aws_ram_resource_share_accepter
+
+Manages accepting a Resource Access Manager (RAM) Resource Share invitation. From a _receiver_ AWS organization, accept an invitation to share resources that were shared by a _sender_ AWS organization. To create a resource share in the _sender_, see the [`aws_ram_resource_share` resource](/docs/providers/aws/r/ram_resource_share.html).
+
+## Example Usage
+
+The invitation can be accepted using the Resource Share Amazon Resource Name (ARN).
+```hcl
+resource "aws_ram_resource_share_accepter" "example" {
```

Comment:
It might be good to take the full test configuration and use it in here for clarity with the `aws_ram_principal_association` resource.

Reply:
added